### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
Added in the heart rate app

### CODE REVIEW

File: CRFHeartRate/CRFHeartRate/Info.plist [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
-	<string>43</string>
+	<string>15</string>
```
Comment (Erin-Mounts) [inline CRFHeartRate/CRFHeartRate/Info.plist]: Curious why this was so much higher than the xcode project's build number.

File: CRFHeartRate/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#  CRFHeartRate
+
+The CRFHeartRate application is designed to be able to send data for heart rate tasks to Sage Bionetworks bridge server as a part of validation studies.
```
Comment (Erin-Mounts) [inline CRFHeartRate/README.md]:
```suggestion
The CRFHeartRate application is designed to be able to send data for heart rate tasks to Sage Bionetworks Bridge server as a part of validation studies.
```

File: README.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ this software without specific prior written permission.
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
 WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> BE LIABLE FOR ANY
+DISCLAIMED. IN NO EVENT SHALL SAGEBIONETWORKS BE LIABLE FOR ANY
```
Comment (Erin-Mounts) [inline README.md]:
```suggestion
DISCLAIMED. IN NO EVENT SHALL SAGE BIONETWORKS BE LIABLE FOR ANY
```

**Review by Erin-Mounts — APPROVED** [review]
couple of nits and a comment/question, otherwise looks good

File: CRFHeartRate/CRFHeartRate/Info.plist (hunk 1 above)
Comment (syoung-smallwisdom) [reply CRFHeartRate/CRFHeartRate/Info.plist]: Whoops, thanks

**Pull request merged**